sample/development/.vscode/launch.json

```diff
@@ -8,7 +8,7 @@
             "host": "${config:roku_ip}",
             "password": "${config:roku_password}",
             "rootDir": "${workspaceFolder}",
-            // "injectRdbOnDeviceComponent": true,
+            "injectRdbOnDeviceComponent": true,
             // "deepLinkUrl": "http://${host}:8060/launch/dev?RunUnitTests=true"
         }
```

Comment:
we need the curly bracket `}`

Reply:
Haha, we have the right `{}` pairs. It's a bug of Github to show those weird red lines here.